File: examples/3_usando_modulos/009_manipulando_texto/ex022_nome_completo/ex022.js
function sayThePropertiesOfName() {

    let inputName = document.querySelector('#input1').value;
    let divAnswer = document.querySelector(".answer");

    if(inputName == "" || !isNaN(inputName)) {

        divAnswer.innerHTML = `<div class="answer error">Informe um valor válido!</div>`;

    } else { 

        divAnswer.innerHTML = "";
        divAnswer.innerHTML += `<div class="answer">Seu nome todo em letras maiúsculas fica assim: </br><strong>${inputName.toUpperCase()}</strong>!</div>`;
        divAnswer.innerHTML += `<div class="answer">Seu nome todo em letras maiúsculas fica assim: </br><strong>${inputName.toLowerCase()}</strong>!</div>`;

        nameNoSpaces = inputName.replace(" ", "");
        divAnswer.innerHTML += `<div class="answer">Seu nome completo tem <strong>${nameNoSpaces.length}</strong> letras (sem contar espaços).</div>`;

        firstName = inputName.split(" ");
        console.log(firstName);
        divAnswer.innerHTML += `<div class="answer">Seu 1° nome é <strong>${firstName[0]}</strong> e tem <strong>${firstName[0].length}</strong> letras.</div>`;
        /* # 4 - Quantas letras tem o primeiro nome. */
        
    }
}

File: python/world_1/examples/3_usando_modulos/009_manipulando_texto/ex022_nome_completo/ex022.py
# Crie um programa que leia o nome completo de uma pessoa e mostre: 
# 1 - O nome com todas as letras maiúsculas;
# 2 - O nome com todas minúsculas;
# 3 - Quantas letras ao todo sem considera os espaços;
# 4 - Quantas letras tem o primeiro nome.

name = str(input('Digite o seu nome: '))

print(f'Seu nome com todas as letras maiúsculas são assim: {name.upper()}')
print(f'Seu nome com todas as letras minúculas são assim: {name.lower()}')

nameNoSpaces = name.replace(" ", "")
print(f'O seu nome sem espaços nenhum fica com {len(nameNoSpaces)} caracteres.')

firsName = name.split()
print(f'Seu primeiro nome é {firsName[0]}! Ele tem {len(firsName[0])} letras.')
